### PR TITLE
Adding cpack include and documenting usage

### PR DESCRIPTION
**Pull request by mbasunov**
Pull request for issue #137 
- Adding cpack include and documenting usage

### CODE REVIEW

**Comment by rodgert** [comment]
Same request, can you squash this to a single commit? I like to keep the commit history to single logically complete changes.

**Comment by mbasunov** [comment]
Done.
Please remember that packaging requires versioning which is done in AzmqCPack.cmake 
`set(CPACK_PACKAGE_VERSION               "1.1.0")
`
That could be reviewed and changed in the future, may be to something like Git does itself.

**Comment by rodgert** [comment]
Good point. I can’t elaborate yet the details, but I expect that within the
next month or so I will have a regular CI, tagging/release and distribution
mechanism for RHEL/CentOS/Fedora in place.

File: AzmqCPack.cmake [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+This library is built on top of ZeroMQ's standard C interface and is
+intended to work well with C++ applications which use the Boost libraries
+in general, and Asio in particular.
+
+The main abstraction exposed by the library is azmq::socket which
+provides an Asio style socket interface to the underlying zeromq socket
+and interfaces with Asio's io_service().  The socket implementation
+participates in the io_service's reactor for asynchronous IO and
+may be freely mixed with other Asio socket types (raw TCP/UDP/Serial/etc.).")
+set(CPACK_PACKAGE_VERSION               "1.1.0")
```
Comment (rodgert) [inline AzmqCPack.cmake]: I will tag a release shortly with these changes, which will be version 1.0.3.  I will update this version to be in sync with the tag version

**Pull request merged**